pkg/common/cns-lib/vsphere/hostsystem.go

```diff
@@ -158,12 +166,16 @@ func (host *HostSystem) GetHostVsanCapacity(ctx context.Context) (*VsanHostCapac
 		return &out, err
 	}
 
-	for _, disk := range disks {
+	for key, disk := range disks {
 		if disk.IsSSD == 1 {
 			// Cache disk doesn't count as capacity.
 			continue
 		}
-		// XXX: Check for health?
+		// Check for disk health
+		if disk.Disk_Health.HealthFlags != 0 {
+			log.Debugf("disk %s is erroneous. Health: %+v", key, disk.Disk_Health)
```

Comment:
shouldn't this be an info level log, so we can help debug things when things fail in the production environment?

Reply:
done